troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+# Troubleshooting Guide for BM IPI Deployments
+
+* [Introduction](#introduction) 
+
+* [Bootstrap VM related issues](#bootstrap-vm-related-issues)
+
+* [Node Related Issues](#node-related-issues)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+# Introduction
+ 
+**BM installation for OCP4:** While deploying BM version of Openshift there are some mandatory requirements which needs to be followed, otherwise it may fail. This guide attempts to cover some of the common scenarios of such events fixing which might help you get a clean cluster. 
+
```

Comment:
I'm not sure we actually need an introduction at all.

Reply:
Removed this Introduction as per suggestion.